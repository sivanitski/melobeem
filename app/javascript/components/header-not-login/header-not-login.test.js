import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import React from "react";
import renderer from "react-test-renderer";

import HeaderNotLogin from "./header-not-login";

Enzyme.configure({
  adapter: new Adapter(),
});

describe("NewIn component", () => {
  it("should render NewIn component", () => {
    const component = renderer.create(<HeaderNotLogin />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
