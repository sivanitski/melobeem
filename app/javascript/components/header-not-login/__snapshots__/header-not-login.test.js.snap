// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewIn component should render NewIn component 1`] = `
<div
  className="header-not-login"
>
  <div
    className="header-not-login__pictures"
  >
    <div
      className="header-not-login__pictures-list__item"
    >
      <img
        alt="Cute Baby"
        src="test-file-stub"
        srcSet="test-file-stub 2x"
      />
    </div>
    <div
      className="header-not-login__pictures-list__item"
    >
      <img
        alt="Cute Baby"
        src="test-file-stub"
        srcSet="test-file-stub 2x"
      />
    </div>
  </div>
  <div
    className="header-not-login__wrapper"
  >
    <div
      className="header-not-login__logo-icon"
    >
      <test-file-stub />
    </div>
    <div
      className="header-not-login__logo-text"
    >
      <test-file-stub />
    </div>
    <div
      className="header-not-login__text text-grey"
    >
      Vote, share with your friends 
      <br />
       complete the tasks, 
      <br />
       there is many ways to win
    </div>
  </div>
  <div
    className="header-not-login__buttons"
  >
    <button
      className="button header-not-login__button"
    >
      Enter competition
    </button>
    <button
      className="button button--facebook header-not-login__button"
    >
      Login with Facebook
    </button>
  </div>
</div>
`;
